scripts/sign-binary.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+CERTIFICATE=${WINDOWS_SIGNING_CERTIFICATE}
+PRIVATE_KEY=${WINDOWS_SIGNING_PRIVATE_KEY}
+WEBSITE_URL=https://github.com/saucelabs/saucectl/
+PROGRAM_DESCRIPTION=saucectl cli
+
+UNIQUENESS=${echo ${RANDOM} | md5sum | cut -b 1-10}
+CRT_PATH=/tmp/certificate-${UNIQUENESS}.crt
+KEY_PATH=/tmp/private-${UNIQUENESS}.key
+TMP_BINARY=${BINARY/saucectl.exe/saucectl-signed.exe}
```

Comment:
From what I can tell, that'd just resolve to `TMP_BINARY` actually pointing to `BINARY`. 🤔 

Maybe all that you want to do is this:
```suggestion
TMP_BINARY=saucectl-signed.exe
```

Which would place the output in the current dir. 
`mv -f ${TMP_BINARY} ${BINARY}` would then do the right thing.

Reply:
Actually it's not resolving `TMP_BINARY` to `BINARY`. 
`BINARY` contains the `path/to/saucectl.exe`, so `TMP_BINARY` will be resolve to `path/to/saucectl-signed.exe`.

I though it would be better to put the signed binary in the same folder as the binary (as there is one output per OS/Arch).